core/storage/user/gae_models.py

```diff
@@ -148,6 +148,20 @@ class UserStatsModel(base_models.BaseMapReduceBatchResultsModel):
     # The impact score.
     impact_score = ndb.FloatProperty(indexed=True)
 
+class UserDashboardStatsModel(base_models.BaseMapReduceBatchResultsModel):
```

Comment:
I was a bit uncertain about this, but then chose to create a new model based on the fact that these stats are shown in a user's dashboard (in contrast to profile page).